content/06.TechnicalValidation.md

```diff
@@ -5,17 +5,17 @@ Each individual dataset also has individual vegetation plots, each provided by i
 We obviously have no direct control on the individual vegetation plots that we provide here in an open access dataset. 
 Yet, each of these vegetation plots are stemming from trained professional botanists, or published scientific work, and are accompanied by detailed information on the sampling protocols used, thus ensuring data quality and reliability.  
 
-Before having been integrated into the sPlot database, each dataset was further checked for consistency and converted to Turboveg format (CIT). 
-During this conversion into a Turboveg format, we checked that all dataset contained the required metadata information and we converted this information to the sPlot database standards, if necessary. 
+Before having been integrated into the sPlot database, each dataset was further checked for consistency and, if having a different format, was converted to a Turboveg 2 database (Hennekens & Schaminée 2001). 
+During this conversion into a Turboveg format, we checked that all datasets contained the required metadata information and we converted this information to the sPlot database standards, if necessary. 
 Furthermore we cross-checked that each plot is located within the geographic scopes of its respective dataset. 
-Finally, we harmonized all the taxonomic names from a dataset, based on the sPlot’s taxonomic backbone (REF). 
+Finally, we harmonized all the taxonomic names from a dataset, based on the sPlot’s taxonomic backbone (Purschke 2017). 
 This backbone matched all the taxonomic names (without nomenclatural authors) from all datasets in sPlot 2.1 and TRY v3.0 (@https://doi.org/10.1111/gcb.14904) to their resolved version based on the Taxonomic Name Resolution Service web application (TNRS version 4.0; @doi:10.1186/1471-2105-14-16; iPlant Collaborative, 2015). 
 This allowed to (1) harmonize all datasets to a common nomenclature, and (2) to link the sPlot database to the TRY database (@https://doi.org/10.1111/gcb.14904). 
 All taxa originally denoted at taxonomic ranks lower than species, were aggregated at species level. 
 Additional detail on the taxonomic resolution is reported in \[@doi:10.1111/jvs.12710\], while a description of the workflow, including R‐code, is available in Purschke (2017).
 
 ## Usage Notes
-\[*something here*\]
+The sPlot Open database can be downloaded from https://www.idiv.de (link to PlantHub). The use of data contained in BioTIME should cite original data citations in addition to the present paper. The data included in the present paper represent the subset of sPlot for which we were able to secure licences for making these data open. The additional studies in sPlot are avalable under sPlot's Governance and Data Property Rules (www.idiv.de/sPlot).
 
 ## Code Availability  
-\[*something here*\]
+The R code used to produce sPlot Open from the sPlot 2.1 database is found here (https://portal.idiv.de/nextcloud/index.php/s/YjMZtwFDwtoefGi). 
```

Comment:
At the moment, this code is in a sister github project:
https://github.com/fmsabatini/sPlotOpen_Code/